### ISSUE TITLE
Searchable Select widget

### ISSUE THREAD

**Issue by alon-sht** [holoviz/panel#3383]
Would be nice to have a searchable select widget
kind of a mix between the regular select and the autocomplete :)

**Comment by xavArtley** [comment]
I wonder if it's not a duplicate of https://github.com/holoviz/panel/issues/3117

**Comment by maximlt** [comment]
Yep it's a duplicate. @alon-sht feel free to chime in in #3117 if you want to give more details about the behaviour you'd expect from this widget.

**Comment by philippjfr** [comment]
Implemented on the panel-material-ui [Select](https://panel-material-ui.holoviz.org/reference/widgets/Select.html#searchable) widget which will be integrated in panel 2.0 but can be used today.
